### ISSUE TITLE
Issue Servodyn .dll files when running openfast on cluster

### ISSUE THREAD

**Issue by tchatte3** [OpenFAST/openfast#483]
Hi openfast team,

I am running a CFD-glue code (Nalu-wind) coupled with openfast. I was using the NREL 5MW baseline case for running the simulations. I am running that on our local cluster. When I run openfast on the headnode as a standalone code it runs fine with the 5MW baseline case giving me consistent results with and without the DISCON.dll controller (i.e.,  with CompSrv = 0 or 1). I compiled .dll libraries using cmake from the local cluster. However, if I run openfast / openfastcpp + CFD-glue code in my cluster by submitting a batch job, it throws up the following error:

``
what():  FAST_InitializeAll:SrvD_Init:BladedInterface_Init:The dynamic library ./5MW_Baseline/../5MW_Baseline/ServoData/DISCON.dll could not be loaded. Check that the file exists in the specified location and that it is compiled for 64-bit applications.
_pmiu_daemon(SIGCHLD): [NID 01696] [c0-1c2s8n0] [Tue Jun 30 12:33:55 2020] PE RANK 0 exit signal Aborted
[NID 01696] 2020-06-30 12:33:55 Apid 164788: initiated application termination
``

Could it be that the .dll compiled for the headnode needs to compiled for the compute node separately as well?

**Comment by rafmudaf** [comment]
@tchatte3 Just as a sanity check, you could remove Nalu from this case and verify that the NREL 5MW turbine runs through your batch script.

Also, are you able to get an interactive session on a node and run again? It may be worth verifying that the path is correct. For example, if your script uses full paths to the OpenFAST executable and `.fst` input file, you may need to `cd` into the case directory first since the controller library path is relative (I'm actually not sure if OpenFAST takes the library-path-input as relative to the shell's `cwd` or relative to the `.fst` file).

I at one point had an environment problem where the runtime-libraries linked to the executable when compiled in the login node weren't available in the compute node. Though I'm not sure if that's the case here because I'd expect the error from OpenFAST before the DISCON DLL.

**Comment by tchatte3** [comment]
@rafmudaf thanks. I actually, tested openfast both as a stand-alone/cpp + CFD glue-code through the job submisision script and both fails related to the issues with the .dll. So, it is definitely openfast problem and not Nalu-wind. Because my architechture in the compute nodes and head node are slightly different, I did compile/build DISCON.dll in the compute nodes through a batch script and test that out. Still the same problem with the .dll.

 Following your advise, I played around with the path: imposing relative as well absolute path to the .dll file, in the Servodyn input; still it is the same problem. It is a bit weird that only the relative paths in Servodyn are so problematic in batch mode submissions, when relative paths in Elastodyn, Aerodyn worked fine.
